doc/reference/dune-project/generate_opam_files.rst

```diff
@@ -20,3 +20,9 @@ generate_opam_files
 With this field, every time one calls Dune to execute some rules (either via
 ``dune build``, ``dune runtest``, or something else), the opam files get
 generated.
+
+To add opam fields not directly supported by ``dune-project``, you can use
+``pkg.opam.template`` files as an escape hatch.
```

Comment:
```suggestion
``<package-name>.opam.template`` files as an escape hatch.
```

I think I would link the reference from `seealso` directly here, because thematically it belongs to the same paragraph.